app/grandchallenge/evaluation/templates/evaluation/result_list.html

```diff
@@ -199,11 +199,50 @@ <h2>Results</h2>
                         "targets": [-1],
                     {% endif %}
                     "searchable": false,
-                    "orderable": false
-                }],
-                ordering: true
+                    "orderable": false,
+                    "visible": true
+                }
+                {% if evaluation_config.extra_results_columns|length > 0 %}
+                ,{
+                    "targets": 'toggable',
+                    "visible": false,
+                }
+                {% endif %}
+                ],
+                ordering: true,
+                autoWidth: false,
+                {% comment %}
+                    Default dom-setting copied from here: https://datatables.net/reference/option/dom
+                {% endcomment %}
+                {% if evaluation_config.extra_results_columns|length > 0 %}
+                dom: "<'row'<'col-5'l><'col-7'f>>" +
+                     "<'row'<'col-sm-12'Btr>>" +
+                     "<'row'<'col-sm-12 col-md-5'i><'col-sm-12 col-md-7'p>>",
+                buttons: [
+                    {
+                        extend: 'colvis',
+                        text: 'Additional metrics',
+                        columns: '.toggable'
+                    },
+                    {
+                        text: 'Show all metrics',
+                        action: function ( e, dt, node, config ) {
+                            if($(node).text().indexOf('Hide') > -1) {
```

Comment:
I do not really like using HTML of the button to determine 'state', maybe use an attribute of a certain element.